torch/jit/supported_ops.py

```diff
@@ -203,7 +203,8 @@ def _get_global_builtins():
         'getattr': 'Attribute name must be a literal string',
         'hasattr': 'Attribute name must be a literal string',
         'isinstance': 'Result is static',
-        'zip': 'Arguments must be lists',
+        'zip': 'Arguments must be a Tensor, list, tuple, dictionary, '
+               ':any:`torch.nn.ModuleList` or :any:`torch.nn.ModuleDict`',
         'enumerate': 'Arguments must be lists',
```

Comment:
same thing for zip applies to enumerate, also you forgot string here

Reply:
I added an iterables section, we can expand on it in a follow up